### ISSUE TITLE
[audit] devel/boost

### ISSUE THREAD

**Issue by jwh** [abyss-os/phase1#40]
it broke

**Comment by jwh** [comment]
```
...failed clang-linux.compile.c++.with-pch bin.v2/libs/math/build/clang-linux-10.0.0/release/threading-multi/visibility-hidden/roundl.o...
clang-linux.compile.c++.with-pch bin.v2/libs/math/build/clang-linux-10.0.0/release/threading-multi/visibility-hidden/tgammal.o
Stack dump:
0.      Program arguments: clang++ -c -x c++ -fvisibility-inlines-hidden -fPIC -pthread -O3 -Wall -fvisibility=hidden -Wno-inline -O2 -Winvalid-pch -DBOOST_ALL_NO_LIB=1 -DBOOST_BUILD_PCH_ENABLED -DBOOST_MATH_TR1_DYN_LINK=1 -DNDEBUG -I. -Ilibs/math/src/tr1 -Xclang -include-pch -Xclang bin.v2/libs/math/build/clang-linux-10.0.0/release/threading-multi/visibility-hidden/../src/tr1/pch.hpp.pth -o bin.v2/libs/math/build/clang-linux-10.0.0/release/threading-multi/visibility-hidden/tgammal.o libs/math/build/../src/tr1/tgammal.cpp
clang-10: error: clang frontend command failed due to signal (use -v to see invocation)
Abyss OS clang version 10.0.0 (https://github.com/abyss-os/phase1.git 8c6d30f400c0e33f4cdf53f79aacf08e095895e8)
Target: mips64-abyss-linux-musl
Thread model: posix
InstalledDir: /usr/bin
clang-10: note: diagnostic msg: PLEASE submit a bug report to https://bugs.llvm.org/ and include the crash backtrace, preprocessed source, and associated run script.
clang-10: note: diagnostic msg:
********************

PLEASE ATTACH THE FOLLOWING FILES TO THE BUG REPORT:
Preprocessed source(s) and associated run script(s) are located at:
clang-10: note: diagnostic msg: /tmp/tgammal-0f27cf.cpp
clang-10: note: diagnostic msg: /tmp/tgammal-0f27cf.sh
clang-10: note: diagnostic msg:

********************
```

**Comment by jwh** [comment]
Fixed on mips64, #38 for mips64el
